src/lib/shells/SlideOver.svelte

```diff
@@ -40,53 +38,51 @@
 			<DialogOverlay class="fixed inset-0 bg-black bg-opacity-75 transition-opacity" />
 		</TransitionChild>
 
-		<div class="fixed inset-0 overflow-hidden">
-			<div class="absolute inset-0 overflow-hidden">
-				<div class="pointer-events-none fixed inset-y-0 right-0 flex max-w-full sm:pl-10">
-					<TransitionChild
-						enter="ease-in-out duration-500 sm:duration-700"
-						enterFrom="translate-x-full"
-						enterTo="translate-x-0"
-						leave="ease-in-out duration-500 sm:duration-700"
-						leaveFrom="translate-x-0"
-						leaveTo="translate-x-full"
-						on:introend={afterOpen}
-						on:outroend={afterClose}
-					>
-						<!-- FIXME: close on click not working for some reason (lib bug or height/placement issue?) -->
-						<div class="pointer-events-auto h-full w-screen max-w-md">
-							<div class="flex h-full flex-col overflow-y-scroll bg-secondary shadow-xl">
-								<div class="flex-1 overflow-y-auto px-4 py-6 sm:px-6">
-									<div class="flex items-start justify-between">
-										<DialogTitle class="text-lg font-medium {title ? '' : 'invisible'}">
+		<div class="absolute inset-0 overflow-hidden">
+			<div class="pointer-events-none fixed inset-y-0 right-0 flex max-w-full">
+				<TransitionChild
+					enter="ease-in-out duration-500 sm:duration-700"
+					enterFrom="translate-x-full"
+					enterTo="translate-x-0"
+					leave="ease-in-out duration-500 sm:duration-700"
+					leaveFrom="translate-x-0"
+					leaveTo="translate-x-full"
+					on:outroend={$onCloseStore}
```

Comment:
5. Finally, execute the store's function whenever we want on the parent component, with our custom content!